ovos_m2v_pipeline/__init__.py

```
@@ -14,236 +11,439 @@
-                match_data={"utterance": utterances[0], "confidence": prob},
+                match_data={"utterance": utterance, "confidence": score},
                 skill_id=skill_id or "ovos-m2v-pipeline",
-                utterance=utterances[0]
+                utterance=utterance,
             )
-            LOG.debug(f"Match candidate: {match}")
-            return match
         return None
 
+    def match_high(
+        self, utterances: List[str], lang: str, message: Message
```

Ruff (0.15.2)

[warning] Unused method argument: `lang`

(ARG002)